pyrmq/publisher.py

```diff
@@ -37,11 +37,19 @@ class Publisher(object):
     queue declares and bindings plus retry logic built for its connection and publishing.
     """
 
-    def __init__(self, exchange_name: str, queue_name: str, routing_key: str, **kwargs):
+    def __init__(
+        self,
+        exchange_name: str,
+        queue_name: Optional[str] = "",
+        routing_key: Optional[str] = "",
```

Comment:
Why?

Reply:
To let users publish without specifying a queue or routing_key.